README.md

```diff
@@ -106,6 +106,133 @@ aio commerce init
 1. Update Storefront to use build.mjs to modify pdp schema
 1. Use this data somehow, ie via slot context, and remove runtime fetch
 
+## Adobe Commerce Integration Starter Kit (Phase 4)
+
+The purpose of this exercise is to demonstrate sending events from Commerce to Adobe Developer App Builder. You will use a simplified version of the Adobe Commerce Integration Starter Kit to configure Commerce and an App Builder project, allowing for the sending of events about saved orders.
+
+### 1. Configure the Starter Kit
+
+1. Login to the Adobe Developer Console at https://developer.adobe.com/console/ and select the **Commerce Extensibility Lab** organization. 
```

Comment:
`Commerce Extensibility Lab` may need to be replaced with the actual Org being used